src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/ethereum/EthereumAdapter.java

```diff
@@ -0,0 +1,705 @@
+/*******************************************************************************
+ * Copyright (c) 2019-2022 Institute for the Architecture of Application System - University of Stuttgart
+ * Author: Ghareeb Falazi
```

Comment:
When you add the contents of the previous class to this one, please don't forget to add yourself as a co-author